client/src/main.rs

```diff
@@ -1059,6 +1059,68 @@ fn main() {
                     Ok(())
                 }),
         )
+        .add_cmd(
+                Command::new("purge-community-ceremony")
+                    .description("purge all history within the provided ceremony index range for the specified community")
+                    .options(|app| {
+                        app.setting(AppSettings::ColoredHelp)
+                            .from_cindex_arg()
+                            .to_cindex_arg()
+
+                    })
+                .runner(|_args: &str, matches: &ArgMatches<'_>| {
+                    let sudoer = AccountKeyring::Alice.pair();
+                    let api = get_chain_api(matches).set_signer(sudoer);
+
+                    let current_ceremony_index = get_ceremony_index(&api);
+
+                    let from_cindex_arg = matches.from_cindex_arg().unwrap_or(0);
+                    let to_cindex_arg = matches.to_cindex_arg().unwrap_or(0);
```

Comment:
I guess we should assert here that `to_cindex_arg >from_index_arg` and give a meaningful error message.

Reply:
true